app/views/subject_rules/_form.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= simple_form_for(@subject_rule, url: [@workflow, @subject_rule]) do |f| %>
+  <%= f.error_notification %>
+
+  <div class="form-inputs">
+    <%= f.input :condition_string, label: 'Condition' %>
```

Comment:
Wouldn't this send it in under the `condition_string` key in the params, and then thus be filtered out?

Reply:
yeah it probably would, i didn't remember that you had updated that in the controller.